src/services/services.ts

```diff
@@ -2912,6 +2919,63 @@ module ts {
 
                 return filteredMembers;
             }
+
+            function isExternalModuleName(token: Node): boolean {
+                if (token && token.parent && token.kind === SyntaxKind.StringLiteral && isInStringOrRegularExpressionOrTemplateLiteral(token)) {

```

Comment:
What is a case where `token.kind === SyntaxKind.StringLiteral` is true, but `isInStringOrRegularExpressionOrTemplateLiteral(token)` is false?